Makefile

```diff
@@ -110,8 +110,9 @@ local-unit-test: $(ENVTEST) ## Run tests.
 
 .PHONY: integration-test
 integration-test: $(ENVTEST) ## Run integration tests.
+	go install github.com/onsi/ginkgo/v2/ginkgo@v2.21.0
```

Comment:
please install the ginko in the .github/workflows/unit-integration-tests.yml similar to what fleet repo does.
https://github.com/Azure/fleet/blob/main/.github/workflows/ci.yml#L47